hoomd/logging.py

```diff
@@ -367,30 +377,29 @@ def log(func=None,
         func (`method`): class method to make loggable. If using non-default
```

Comment:
Can the first paragraph of the documentation for this class not start with "for users", i.e. it should read "This should be used with ..."

Reply:
Done.